OrbitGl/App.cpp

```diff
@@ -954,13 +954,14 @@ bool OrbitApp::GetSamplingEnabled() { return GParams.m_TrackSamplingEvents; }
 
 //-----------------------------------------------------------------------------
 
-void OrbitApp::OnProcessSelected(uint32_t pid) {
-  thread_pool_->Schedule([pid, this] {
+void OrbitApp::OnProcessSelected(uint32_t pid,
+                                 const std::string& process_name) {
```

Comment:
also you can probably use dataview to get process_data by pid and get name from there.

Reply:
You mean retrieve the `ProcessesDataView` with `GetOrCreateDataView`?

Reply:
> You mean retrieve the `ProcessesDataView` with `GetOrCreateDataView`?

data_manager_->GetProcessById() - there is a map which I added for this reason - I have not yet added the method.

Reply:
Done, I had to add a couple of methods, please have a look.